ote/test/clj/ote/transit_changes/detection_test.clj

```diff
@@ -84,31 +173,33 @@
          {route-name ["h1" "h2" "h3" "h4" "h5" nil nil]} ;; 22.10.
          {route-name ["h1" "h2" "h3" "h4" "h5" nil nil]} ;; 29.10.
          {route-name ["h1" "h2" "h3" "h4" "h5" nil nil]} ;; 5.11.
-         {route-name ["h1" nil nil nil nil nil nil]} ; 6 day run
-         {route-name [nil nil nil nil nil nil nil]} ; 7 days
-         {route-name [nil nil nil nil "h5" nil nil]} ; 4 days => sum 17
+         {route-name ["h1" nil nil nil nil nil nil]}     ;; Starts 13.11, 6 day run
+         {route-name [nil nil nil nil nil nil nil]}      ;;        7 days
+         {route-name [nil nil nil nil "h5" nil nil]}     ;; Ends 30.11, 4 days => sum 17
          {route-name ["h1" "h2" "h3" "h4" "h5" nil nil]}
          {route-name ["h1" "h2" "h3" "h4" "h5" nil nil]}
          {route-name ["h1" "h2" "h3" "h4" "h5" nil nil]}))
 
 (deftest no-traffic-run-weekdays-is-detected
-  ;; Test that traffic that has normal "no-traffic" days (like no traffic on weekends)
-  ;; is still detected.
-  (is (= {:no-traffic-start-date (d 2018 11 13)
-          :no-traffic-end-date (d 2018 11 30)}
-         (-> (detection/route-weeks-with-first-difference-new test-no-traffic-run-weekdays)
-             first
-             (select-keys [:no-traffic-start-date :no-traffic-end-date])))))
+  (testing "Ensure no-traffic run is reported and normal no-traffic weekends are not. "
+    (is (= {:route-key route-name
+            :starting-week {:beginning-of-week (d 2018 10 15) :end-of-week (d 2018 10 21)}
+            :no-traffic-change 17
+            :no-traffic-start-date (d 2018 11 13)
+            :no-traffic-end-date (d 2018 11 30)}
+           (-> (detection/route-weeks-with-first-difference-new test-no-traffic-run-weekdays)
+               first
+               (select-keys select-keys-detect-changes-for-all-routes))))))
 
 (def no-traffic-run-full-detection-window
```

Comment:
When refactorig def name could be also changed to ->data-no-traffic-run-full-detection-window